test/blackbox/common/DDSBlackboxTestsDiscovery.cpp

```diff
@@ -168,3 +172,65 @@ TEST(DDSDiscovery, AddDiscoveryServerToList)
     server_2.wait_discovery(std::chrono::seconds::zero(), 2, true);
 }
 
+/**
+ * This test checks the addition of network interfaces at run-time.
```

Comment:
You'll need to update the description. Also, some steps in the tests are commented and some aren't. Same goes for the RTPS test